basket/base/decorators.py

```diff
@@ -1,13 +1,60 @@
 import functools
+from time import time
 
 from django.conf import settings
 
 from django_statsd.clients import statsd
-from rq.decorators import job as rq_job
+from rq.decorators import job
 
 from basket.base.rq import enqueue_kwargs, get_queue
 
 
+class rq_job(job):
+    def __call__(self, f):
+        @functools.wraps(f)
+        def delay(*args, **kwargs):
+            if isinstance(self.queue, str):
+                queue = self.queue_class(name=self.queue, connection=self.connection)
+            else:
+                queue = self.queue
+
+            depends_on = kwargs.pop("depends_on", None)
+            job_id = kwargs.pop("job_id", None)
+            at_front = kwargs.pop("at_front", False)
+
+            if not depends_on:
+                depends_on = self.depends_on
+
+            if not at_front:
+                at_front = self.at_front
+
+            # Note: This line is the only addition to the parent class.
+            # Copied from rq/decorators.py in v1.15.1.
+            self.meta.update(start_time=time())
```

Comment:
No since the `delay` call is attached to the function afterwards. I tried that first but the decorator wrapping kinda gets in the way.

Reply:
> No since the `delay` call is attached to the function afterwards. I tried that first but the decorator wrapping kinda gets in the way.

Of course 🤦 dumb question

Reply:
There are no dumb questions :)